### ISSUE TITLE
Delete SPV file and resync button does not work

### ISSUE THREAD

**Issue by jmeemiller** [bisq-network/bisq#1078]
**Platform:** Bisq client 0.6.1, Java 8 update 131, Windows 7 64-bit Service Pack 1

When you click on the "Delete SPV file and resync" button under "Settings->Network Info", it displays an error dialogue (see attached) and fails to delete the file.

![capture](https://user-images.githubusercontent.com/34354426/33810245-14b0a658-de0b-11e7-9115-37365c5ca366.PNG)

No message is added to the bisq.log file when clicking the button.

**Comment by ghost** [comment]
It also reproducible by me on Win7 every time but without any log records, silently. Occured in `NetworkServiceView:249`:

```java
        reSyncSPVChainButton.setOnAction(event -> {
            if (walletsSetup.reSyncSPVChain()) {
                new Popup<>().feedback(Res.get("settings.net.reSyncSPVSuccess"))
                        .useShutDownButton()
                        .actionButtonText(Res.get("shared.shutDown"))
                        .onAction(() -> {
                            preferences.setResyncSpvRequested(true);
                            UserThread.runAfter(BisqApp.shutDownHandler::run, 100, TimeUnit.MILLISECONDS);
                        })
                        .hideCloseButton()
                        .show();
            } else {
>>>                new Popup<>().error(Res.get("settings.net.reSyncSPVFailed")).show();
            }
        });
```

**Comment by ghost** [comment]
The problem is the SPV chain file is locked by Windows:

```
    public boolean reSyncSPVChain() {
        try {
>>>            return new File(walletDir, SPV_CHAIN_FILE_NAME).delete();
        } catch (Throwable t) {
            log.error(t.toString());
            t.printStackTrace();
            return false;
        }
    }
```

`delete` method simply returns `false`. Looks like it locked by `bitcoinj` library in `SPVBlockStorage` class.

![image](https://user-images.githubusercontent.com/39793/33811126-b61dfa6a-de0e-11e7-8906-f1f7422afde2.png)

**Comment by regnarock** [comment]
After https://github.com/bisq-network/support/issues/61, I wanted to click on "Delete SPV file and resync" button under "Settings->Network Info" as asked and had the same error than @jmeemiller mentioned.

There doesnt seem to have anything in the log. Let me know if I can provide additional informations.

**Comment by ManfredKarrer** [comment]
I just fixed that for the 0.6.5 release which got deployed just now.
https://github.com/bisq-network/exchange/releases/tag/v0.6.5

**Comment by cbeams** [comment]
Closed as duplicate of #1289.
